### ISSUE TITLE
Run CIS Docker Bench for Trivy image and fix the warnings

### ISSUE THREAD

**Issue by danielpacak** [aquasecurity/harbor-scanner-trivy#27]

**Comment by skurtzemann** [comment]
Do you mean the section 4 `Container Images and Build File Configuration` of the last CIS Docker Bench (v1.2.0) ?

If a perform a analyse with  `docker/docker-bench-security` I got 
```
[INFO] 4 - Container Images and Build File                                                                                                                                                                                                                                         
[WARN] 4.1  - Ensure a user for the container has been created                                                                                                                                                                                                                     
[WARN]      * Running as root: vibrant_ganguly                                                                                                                                                                                                                                     
[NOTE] 4.2  - Ensure that containers use trusted base images                                                                                                                                                                                                                       
[NOTE] 4.3  - Ensure unnecessary packages are not installed in the container                                                                                                                                                                                                       
[NOTE] 4.4  - Ensure images are scanned and rebuilt to include security patches                                                                                                                                                                                                    
[WARN] 4.5  - Ensure Content trust for Docker is Enabled                                                                                                                                                                                                                           
[WARN] 4.6  - Ensure HEALTHCHECK instructions have been added to the container image                                                                                                                                                                                               
[WARN]      * No Healthcheck found: [aquasec/harbor-scanner-trivy:0.1.0-rc2]                                                                                                                                                                                                       
[PASS] 4.7  - Ensure update instructions are not use alone in the Dockerfile                                                                                                                                                                                                       
[NOTE] 4.8  - Ensure setuid and setgid permissions are removed in the images                                                                                                                                                                                                       
[INFO] 4.9  - Ensure COPY is used instead of ADD in Dockerfile                                                                                                                                                                                                                     
[INFO]      * ADD in image history: [aquasec/harbor-scanner-trivy:0.1.0-rc2]                                                                                                                                                                                                       
[NOTE] 4.10  - Ensure secrets are not stored in Dockerfiles                                                                                                                                                                                                                        
[NOTE] 4.11  - Ensure verified packages are only Installed 
```

**Comment by danielpacak** [comment]
> Do you mean the section 4 `Container Images and Build File Configuration` of the last CIS Docker Bench (v1.2.0) ?
> 
> An analyse with `docker/docker-bench-security` returns :
> 
> ```
> [INFO] 4 - Container Images and Build File                                                                                                                                                                                                                                         
> [WARN] 4.1  - Ensure a user for the container has been created                                                                                                                                                                                                                     
> [WARN]      * Running as root: vibrant_ganguly                                                                                                                                                                                                                                     
> [NOTE] 4.2  - Ensure that containers use trusted base images                                                                                                                                                                                                                       
> [NOTE] 4.3  - Ensure unnecessary packages are not installed in the container                                                                                                                                                                                                       
> [NOTE] 4.4  - Ensure images are scanned and rebuilt to include security patches                                                                                                                                                                                                    
> [WARN] 4.5  - Ensure Content trust for Docker is Enabled                                                                                                                                                                                                                           
> [WARN] 4.6  - Ensure HEALTHCHECK instructions have been added to the container image                                                                                                                                                                                               
> [WARN]      * No Healthcheck found: [aquasec/harbor-scanner-trivy:0.1.0-rc2]                                                                                                                                                                                                       
> [PASS] 4.7  - Ensure update instructions are not use alone in the Dockerfile                                                                                                                                                                                                       
> [NOTE] 4.8  - Ensure setuid and setgid permissions are removed in the images                                                                                                                                                                                                       
> [INFO] 4.9  - Ensure COPY is used instead of ADD in Dockerfile                                                                                                                                                                                                                     
> [INFO]      * ADD in image history: [aquasec/harbor-scanner-trivy:0.1.0-rc2]                                                                                                                                                                                                       
> [NOTE] 4.10  - Ensure secrets are not stored in Dockerfiles                                                                                                                                                                                                                        
> [NOTE] 4.11  - Ensure verified packages are only Installed 
> ```

The most important one is 4.1 -> there's no reason to run adapter as root.

**Comment by danielpacak** [comment]
Running as non root was solved by #66
